dist/init/systemd/rkt-api-tcp.socket

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=rkt api service socket
+PartOf=rkt-api.service
+
+[Socket]
+ListenStream=15441
```

Comment:
Right we should add socket activation support to the API service. Similar plumbing is already in place for the metadata service: https://github.com/coreos/rkt/blob/master/rkt/metadata_service.go#L572